bindings/2.206/GeometryDash.bro

```diff
@@ -12259,25 +12259,9 @@ class PlayLayer : GJBaseGameLayer, CCCircleWaveDelegate, CurrencyRewardDelegate,
 	void delayedFullReset();
 	void delayedResetLevel() = win 0x395700;
 	void fullReset() = win 0x395600;
-	float getCurrentPercent() = win inline {
```

Comment:
getCurrentPercent() is inlined on imac, so would be great if you didn't remove the definition entirely

Reply:
mb, is that for intel macs, ASi or both?

Reply:
not sure about m1 macs, but i was looking for intel macs bindings an hour ago which had this function inlined

Reply:
should be sorted now